go/vt/vttablet/tabletserver/vstreamer/planbuilder.go

```diff
@@ -45,6 +45,7 @@ type Plan struct {
 // ColExpr represents a column expression.
 type ColExpr struct {
 	ColNum int
+	Vindex vindexes.Vindex
 	Alias  sqlparser.ColIdent
 	Type   querypb.Type
 }
```

Comment:
Can you add comments for each field here? It isn't obvious that Vindex will be set only for `keyspace_id` func as of now.

Reply:
I'll update these on the next PR, because I've reorganized these a bit to accommodate multi-column vindexes.